### ISSUE TITLE
Release TwistedConch-0.8

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2359]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2359|
|Type|task|
|Created|2007-01-06 19:12:24Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2359 2359
type__task task
reporter__radix radix
priority__normal normal
milestone__Conch_0_8 Conch-0.8
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168110744000000 1168110744000000
changetime__1168134846000000 1168134846000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix set owner to @ralphm|
|-|-|

ok, ready for review at releases/twisted-conch-0.8-2359

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to z3p|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @radix|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [19226]) Merge releases/twisted-conch-0.8-2359

Author: release automation, radix
Reviewer: exarkun
Fixes [#2359](https://github.com/twisted/twisted/issues/2359)

Version numbers and NEWS from release of conch 0.8.0

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
